swagger/openemr-api.json

```diff
@@ -0,0 +1,384 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "title": "OpenEMR API",
+        "version": "6.0.0"
```

Comment:
hi @bradymiller , is the version `6.1.0`?

Reply:
yep, i thought i changes that, but will make sure (this PR is still a ways off; still trying to get the oauth2 process to work from within swagger to allow easy testing of the api and then will full build out the api documentation)

Reply:
really cool development, thank you @adunsulag & @bradymiller

Reply:
oh actually, need to bring in most recent openemr-api.json/openemr-api.yaml (forgot to do that) to show most recent changes in the documentation.